### PR TITLE
Added readable error output

### PR DESCRIPTION
**Pull request by nikedlab**

### CODE REVIEW

**Comment by ghostsquad** [comment]
@nikedlab why add more steps to find out the error? Why not just include the request body in the error?

File: jira.go [hunk 1]
```diff
@@ -267,7 +267,20 @@ func CheckResponse(r *http.Response) error {
...
+	if jsonErr := json.NewDecoder(r.Body).Decode(&jiraError); jsonErr != nil {
+		fmt.Println("Response JSON decoding failed! Error: " + jsonErr.Error())
+	}
+
+	var messages string
+	for _, msg := range jiraError.Errors {
+		messages +=  msg + "\n"
+	}
+
+	err := fmt.Errorf("Request failed. Please analyze the request body for more details. Status code: %d\n%s",
```
Comment (ghostsquad) [inline jira.go]: @nikedlab why add more steps to find out the error? Why not just include the request body in the error?

**Comment by ghostsquad** [comment]
This PR is getting stale. I'll be closing in next week if no one is looking at it.

File: jira.go [hunk 2]
```diff
@@ -267,7 +267,20 @@ func CheckResponse(r *http.Response) error {
 		return nil
 	}
 
-	err := fmt.Errorf("Request failed. Please analyze the request body for more details. Status code: %d", r.StatusCode)
+	var jiraError JiraError
+	if jsonErr := json.NewDecoder(r.Body).Decode(&jiraError); jsonErr != nil {
+		fmt.Println("Response JSON decoding failed! Error: " + jsonErr.Error())
```
Comment (ghostsquad) [inline jira.go]: libraries shouldn't print directly to stdout.

**Comment by andygrunwald** [comment]
Stale PR.
If this Pull Request receives new activity, I will reopen it.
Thanks for taking the time.

**Pull request closed without merging**